compiler/compiler.go

```diff
@@ -183,6 +184,11 @@ func WritePkgCode(pkg *Archive, dceSelection map[*Decl]struct{}, minify bool, w
 			panic(err)
 		}
 	}
+	for _, d := range pkg.JSDecls {
```

Comment:
Would it make sense to add a `(function() { ... })()` wrapper around it, so it can't interfere with the generated code?